src/ui.rs

```diff
@@ -442,6 +536,118 @@ impl App {
                         _ => {}
                     }
                 }
+                InputMode::QuickReply { target_peer } => {
+                    let target_peer = target_peer.clone(); // Clone to avoid borrow checker issues
+                    match key.code {
+                        KeyCode::Enter => {
+                            let message = self.input.trim().to_string();
+                            self.input.clear();
+                            self.input_mode = InputMode::Normal;
+                            if !message.is_empty() {
+                                let cmd = format!("msg {} {}", target_peer, message);
+                                self.add_to_log(format!("> {}", cmd));
+                                return Some(AppEvent::Input(cmd));
+                            }
+                        }
+                        KeyCode::Esc => {
+                            self.input_mode = InputMode::Normal;
+                            self.input.clear();
+                            self.add_to_log("❌ Quick reply cancelled".to_string());
+                        }
+                        KeyCode::Char(c) => {
+                            if key.modifiers.contains(KeyModifiers::CONTROL) {
+                                if c == 'c' {
+                                    self.input_mode = InputMode::Normal;
+                                    self.input.clear();
+                                    self.add_to_log("❌ Quick reply cancelled".to_string());
+                                }
+                            } else {
+                                self.input.push(c);
+                            }
+                        }
+                        KeyCode::Backspace => {
+                            self.input.pop();
+                        }
+                        _ => {}
+                    }
+                }
+                InputMode::MessageComposition { target_peer, lines, current_line } => {
+                    let target_peer = target_peer.clone(); // Clone to avoid borrow checker issues
+                    let mut new_lines = lines.clone();
+                    let mut new_current = current_line.clone();
+                    
+                    match key.code {
+                        KeyCode::Enter => {
+                            if key.modifiers.contains(KeyModifiers::CONTROL) {
+                                // Ctrl+Enter to send the multi-line message
+                                if !new_current.trim().is_empty() {
```

Comment:
The CTRL+Enter is not working.  It just adds new lines.  See attached image

<img width="1292" height="50" alt="Image" src="https://github.com/user-attachments/assets/30a637ce-7784-4e0b-92e3-2852648f83f2" />